### DISCUSSION TITLE
Code Completion not working

### DISCUSSION THREAD

**Discussion by francwalter** [dbeaver/dbeaver#19462] [Q&A]
Hello
I have DBeaver Version 23.0.0.202303040621 on Ubuntu 22.04
I have set Code Completion in Preferences as this:

![image](https://user-images.githubusercontent.com/927269/227161361-f6dcc2a1-0e9d-4fc5-a137-d2b8ed140538.png)

But when I write some SQL, only the first word is proposed as code completion:

![image](https://user-images.githubusercontent.com/927269/227161462-054c4680-20c1-4511-965f-e41f294bcb1c.png)

The following code is not completed or suggested, it says "no proposals".
In my screenshot I expect the keyword "INTO" as proposal, which is very often used after "INSERT" :smirk: 

I guess I have something misconfigured, but I dont find it.

Somebody has an idea?
Could it lastly be a bug?
I have the 23.0 on Windows 10 (64) and there it is working.

And is there a way to have UPPER CASE in keywords even if I dont take the proposals?
So if I write "select", the code completion corrects it to "SELECT" without pressing CTRL+SPACE (if it is suggested)?

Thanks for hints.
frank

**Comment by LonwoLonwo** [comment]
https://github.com/dbeaver/dbeaver/issues/18316 INTO after INSERT case fixed here, will be available in the next DBeaver release.
